src/app/components/home/home.component.html

```diff
@@ -35,7 +35,7 @@
             <!-- city council subcategory and its sub properties -->
             <ng-template #elseBlockContainer>
               <mat-expansion-panel *ngFor="let candidate of candidateType.value.candidates | keyvalue; let j=index" [expanded]="councilDistrictStep===j"
-                (opened)="setCouncilDistrictStep(j)" class="mat-elevation-z0" [routerLink]="'/'+ candidate.key">
+                (opened)="setCouncilDistrictStep(j)" class="mat-elevation-z0" [routerLink]="['city-council/district', candidate.key]">
```

Comment:
can we make the first part of the routerlink value a variable that we get from candidate list?

Reply:
or we could have it as a constant since city-council/district is being used throughout the solution.

Reply:
Good idea. Thanks.

Reply:
np

Reply:
I don't see a good place add city-council/district into the list without creating a workaround. For now I will leave it as is. I am working on updating the candidates.json file and the data structure created from it. This should allow for the path to be read from a variable and make it easier to unify the 3 types of candidate components.

Reply:
ok